src/Microsoft.Azure.WebJobs.Host/Common/CommonCollectorBinding.cs

```diff
@@ -6,7 +6,7 @@
 using Microsoft.Azure.WebJobs.Host.Bindings;
 using Microsoft.Azure.WebJobs.Host.Protocols;
 
-namespace Microsoft.Azure.WebJobs.ServiceBus
+namespace Microsoft.Azure.WebJobs.Host
```

Comment:
I think all of these binding specific types should be in their own namespace so they don't pollute the top level namespace. Perhaps the existing "Bindings" namespace.

Also, recommend moving these to the existing Bindings folder, rather than a "Common" folder.